tensorflow/core/kernels/sparse/mat_mul_op.cc

```diff
@@ -539,7 +539,11 @@ class CSRMatMulGPUOp : public CSRMatMulOp<GPUDevice, T> {
 
     const GPUDevice& d = ctx->eigen_device<GPUDevice>();
 
-    if (b_outer_dim == 1) {
+    if (b_outer_dim == 1
+#if TENSORFLOW_USE_ROCM
+        && (!this->transpose_a_ && !this->conjugate_a_)
```

Comment:
Is `functor::CSRSparseMatrixMatVec<GPUDevice, T>` not working on ROCm therefore to macro it out in this branch?

Reply:
re-word this to something like

```
bool rocm_specific_check = true;
#if TENSORFLOW_USE_ROCM
// add comment here explaining why this check is needed for ROCm
// guessing we are trying to avoid/workaround some path for which we do not have ROCm support.
rocm_specific_check = (!this->transpose_a_ && !this->conjugate_a_);
#endif

if ((b_outer_dim == 1) && rocm_specific_check)
```

Reply:
@jerryyin The method is not working if transpose is requested. It ends up here https://github.com/ROCmSoftwarePlatform/rocSPARSE/blob/develop/library/src/level2/rocsparse_csrmv.hpp#L1112

Reply:
@ekuznetsov139 Thanks for the details. Could you help document it either as an individual issue or in the [existing rocBLAS one](https://github.com/ROCmSoftwarePlatform/tensorflow-internal/issues/7). Just so that we keep track of those little feature in-parities.